.changeset/dirty-owls-remain/changes.md

```diff
@@ -0,0 +1 @@
+Add type for useTheme hook in emotion-theming
```

Comment:
could you mention here that this is about TS types?

Reply:
Sure, I've rephrased this to be clearer. Can you take a look?